setup.cfg

```diff
@@ -1,11 +1,14 @@
+[aliases]
+test=pytest
```

Comment:
I'd like to add the arguments here, i.e.

```
test=pytest --flake8 --cov=documenteer
```

I've been dropping the `addopts` in my other packages because I find it prevents me from using pytest in ad hoc ways, like running a single test module or function.

Reply:
The semantics here are to setup aliases of existing `setup.py` sub-commands.  This declaration aliases `test` -> `pytest` and, AFAIK, the sub-commands do not accept parameters. Eg.
```
 python setup.py pytest --flake8
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --flake8 not recognized
```

Reply:
ah, bummer.